TeamCode/src/main/java/org/firstinspires/ftc/teamcode/actuators/MecanumDrive.java

```diff
@@ -1,49 +1,46 @@
 package org.firstinspires.ftc.teamcode.actuators;
 
-/**
- * Created by LBYPatrick on 2017/10/18.
- */
-
 import com.qualcomm.robotcore.hardware.DcMotor;
 
-import org.firstinspires.ftc.teamcode.LinearOp;
-import org.firstinspires.ftc.teamcode.Statics;
+/**
+ * Created by LBYPatrick on 10/27/2017.
+ */
 
-public class MecanumDrive extends LinearOp{
+final public class MecanumDrive {
```

Comment:
why final?

Reply:
Reference won't get changed for this class so final could make the program faster without any cost :p

Reply:
Just be careful......how do you know it won't be change? For example, a weird motor or others would mess up the performance......